tests/SmokeTest.php

```diff
@@ -3,16 +3,20 @@
 use PHPUnit\Framework\TestCase;
 
 /**
- * @licence GNU GPL v2+
+ * @license GPL-2.0-or-later
  * @author Jeroen De Dauw < jeroendedauw@gmail.com >
  */
 class SmokeTest extends TestCase {
-
+	// Ignoring CS Warnings for the @covers rule as this file does not cover any php classes
+	// phpcs:disable MediaWiki.Commenting.MissingCovers.MissingCovers
```

Comment:
I think there’s also a `@coversNothing` annotation you can put in the class documentation instead of disabling the rule; that seems nicer to me.

Reply:
…or in the method documentation, I guess, sure.